packages/block-editor/src/components/block-lock/style.scss

```diff
@@ -53,3 +53,15 @@
 .block-editor-block-lock-modal__actions {
 	margin-top: $grid-unit-30;
 }
+
+.block-editor-block-lock-toolbar {
+	.components-button.has-icon {
+		min-width: $button-size-small + $grid-unit-15 !important;
+		padding-left: 0 !important;
+
+		&:focus::before {
+			left: 0 !important;
+			right: $grid-unit-15 !important;
```

Comment:
I'm not a huge fan of using all these `!important` here. The mover's code has the following comment.

> // The !important modifier should be removed when https://github.com/WordPress/gutenberg/issues/24898 refactors the spacing grid.

@jasmussen, do you think we can remove them now as I see that issue is closed?

Reply:
Yeah the whole block toolbar deserves a refactor that makes it a single bordered flex container with gap and separators instead of all this funky first-child/last-child stuff. 

And to answer your question: if you can remove it, and it looks good, definitely go for it. I just have this feeling that the block toolbar refactor may be a biggish thing necessary in order to do it right.

Reply:
Currently, I've to use them like block movers. But this toolbar item will also benefit from future refactor, so I guess it okay to have them for now 😄